tests/unit/test_views.py

```diff
@@ -293,6 +293,13 @@ def test_with_classifiers(self, monkeypatch, db_request, page):
         classifier2 = ClassifierFactory.create(classifier="foo :: baz")
         classifier3 = ClassifierFactory.create(classifier="fiz :: buz")
 
+        project = ProjectFactory.create()
+        release1 = ReleaseFactory.create(project=project)
+        release1.created = datetime.date(2011, 1, 1)
+        release1._classifiers.append(classifier1)
+        release1._classifiers.append(classifier2)
+        release1._classifiers.append(classifier3)
```

Comment:
Should I left `classifier3` alone and add an assert to test that it's not in `search(db_request)['available_filters']`? Right now, we are not actually testing the new behavior.